### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'name'

### ISSUE THREAD

**Issue by JackWolf77** [AyushSehrawat/Economy-Bot#15]
Hi! I hope for your help. So the code you shared works really well and I'm thankful you shared it, but I have a problem with the leaderboard command. Whenever I use it, I get this error: **discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'name'**. It's in the line 385, where **name = member.name** . Do you know any solutions to this?

**Comment by AyushSehrawat** [comment]
Well it worked for me when i made, please verify code again , or you can remake leaderboard with another db like mongo or sql

**Comment by AyushPande123** [comment]
yeah same error I got. so when I deleted all the "MainBank.json" data then it perfectly worked for me
